client_v25.go

```diff
@@ -251,6 +258,10 @@ func (d *Dgraph) Close() {
 
 // signInUser logs the user in using the provided username and password.
 func (d *Dgraph) signInUser(ctx context.Context, username, password string) error {
+	if d.useV24 {
+		return d.login(ctx, username, password, 0)
```

Comment:
From what I can see, this path doesn't update `d.jwt.RefreshJwt` with the resp. Is that a problem?

Reply:
It does right before it returns
```
return proto.Unmarshal(resp.Json, &d.jwt)
```